layouts/community/ergodox/replicaJunction/keymap.c

```diff
@@ -1,427 +1,382 @@
-           KC_M    ,KC_N    ,KC_E_LT ,KC_I_LT ,KC_O    ,KC_QUOT,
-//--------|--------|--------|--------|--------|--------|-------|
-  KC_LOCK ,KC_K    ,KX_H_MT ,KX_COMT ,KX_DOMT ,KX_SLMT ,KC_RSFT,
-//--------|--------|--------|--------|--------|--------|-------|
-                    KC_LALT ,KC_MINS ,KC_QUOT ,KC_EQL  ,TT_NUM ,
-//--------|--------|--------|--------|--------|--------|-------|
-  KC_LEFT ,KC_RGHT ,
-//--------|--------|
-  KC_UP   ,
-//--------|--------|--------|
-  KC_DOWN ,KX_NALT ,KX_SPAC
-//--------|--------|--------|
+#define BSP_SYM LT(L_SYMBOLS, KC_BSPC)
+#define SPC_NAV LT(L_NAVIGATION, KC_SPC)
 
-),
+#define MO_FN MO(L_FN)
+#define MO_MAC MO(L_MACROS)
+#define MO_SLED MO(L_SYSLEDS)
+
+#define TG_GAME TG(L_GAMING)
+
+#define ESC_SFT  LSFT_T(KC_ESC)
+#define TAB_SFT  RSFT_T(KC_TAB)
 
 
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
 
-[L_QWERTY] = LAYOUT_ergodox(
-//--------|--------|--------|--------|--------|--------|--------|
-  _______ ,_______ ,_______ ,_______ ,_______ ,_______ ,_______ ,
-//--------|--------|--------|--------|--------|--------|--------|
-  _______ ,KC_Q    ,KC_W    ,KC_E    ,KC_R    ,KC_T    ,_______ ,
-//--------|--------|--------|--------|--------|--------|--------|
-  _______ ,KC_A    ,KC_S    ,KC_D    ,KC_F    ,KC_G    ,
-//--------|--------|--------|--------|--------|--------|--------|
-  _______ ,KC_Z    ,KC_X    ,KC_C    ,KC_V    ,KC_B    ,_______ ,
-//--------|--------|--------|--------|--------|--------|--------|
-  _______ ,_______ ,_______ ,_______ ,_______ ,
-//--------|--------|--------|--------|--------|--------|--------|
-                                               _______ ,_______ ,
-//                                            |--------|--------|
-                                                        _______ ,
-//                                   |--------|--------|--------|
-                                      _______ ,_______ ,_______ ,
-//                                   |--------|--------|--------|
-
-//--------|--------|--------|--------|--------|--------|--------|
-  _______ ,_______ ,_______ ,_______ ,_______ ,_______ ,KC_EQL  ,
-//--------|--------|--------|--------|--------|--------|--------|
-  _______ ,KC_Y    ,KC_U    ,KC_I    ,KC_O    ,KC_P    ,_______ ,
-//--------|--------|--------|--------|--------|--------|--------|
-           KC_H    ,KC_J    ,KC_K    ,KC_L    ,KC_SCLN ,KC_QUOT ,
-//--------|--------|--------|--------|--------|--------|--------|
-  KC_LOCK ,KC_N    ,KC_M    ,KC_COMM ,KC_DOT  ,KC_SLSH ,KC_RSPC ,
-//--------|--------|--------|--------|--------|--------|--------|
-                    KC_RALT ,KC_MINS ,KC_QUOT ,KC_EQL  ,TT_NUM  ,
-//--------|--------|--------|--------|--------|--------|--------|
-  KC_LEFT ,KC_RGHT ,
-//--------|--------|
-  KC_UP   ,
-//--------|--------|--------|
-  KC_DOWN ,KX_NALT ,KX_SPAC
-//--------|--------|--------|
+[L_BASE] = LAYOUT_ergodox(
```

Comment:
`LAYOUT_ergodox_pretty`?

Reply:
I actually prefer this format - the smaller width makes it easier to use a vertical split in an editor. That way, I can have the keymap up on the left tab and a userspace file on the right.

Is `LAYOUT_ergodox_pretty` an expected standard? If so, I'll change my code, but if it's still optional, I'd rather leave it like this.